### PR TITLE
Integrate browserslist and fix tox

### PR DESCRIPTION
**Pull request by FinalAngel**
allows for full spectrum of browser tests and fix tox

### CODE REVIEW

**Comment by vxsx** [comment]
:+1:

File: aldryn_events/tests/frontend/base.conf.js [hunk 1]
```diff
@@ -7,28 +7,17 @@
...
+var b2s = require('browserslist-saucelabs');
+
 module.exports = {
     formatTaskName: function formatTaskName(browserName) {
         return [
             'Test', browserName, 'for',
             process.env.TRAVIS_REPO_SLUG,
-            (process.env.TRAVIS_PULL_REQUEST ? 'pull request #' + process.env.TRAVIS_PULL_REQUEST : ''),
+            (process.env.TRAVIS_PULL_REQUEST !== 'false' ? 'pull request #' + process.env.TRAVIS_PULL_REQUEST : ''),
             'build #' + process.env.TRAVIS_BUILD_NUMBER
```
Comment (vxsx) [inline aldryn_events/tests/frontend/base.conf.js]: can you change it to TRAVIS_JOB_NUMBER ? would be cleaner on saucelabs logs

**Comment by vxsx** [comment]
@Marketionist tests are failing on local machine because apphook config is not specified

**Comment by Marketionist** [comment]
What exactly do you mean by apphook config is not specified @vxsx ?

**Comment by vxsx** [comment]
screen shot 2015-07-10 at 13 15 23

this is the screenshot i made when was testing locally - after the second it() block it doesn't show the success message, instead it shows this

**Comment by FinalAngel** [comment]
LGTM :dancer:

**Comment by FinalAngel** [comment]
@vxsx let's discuss this quickly in the morning on the actions

**Pull request merged**